src/Distributions.jl

```diff
@@ -18,7 +18,7 @@ import StatsBase: kurtosis, skewness, entropy, mode, modes,
                   params, params!
 
 import PDMats: dim, PDMat, invquad
-import GSL: sf_gamma_inc_P
+import Rmath: pgamma
```

Comment:
I think it would be better to use `cdf(Gamma(...), ...)` instead of `pgamma` directly. It effectively calls the same thing but uses the Distributions API.

Reply:
I swapped the pgamma call as you suggested, the call is `cdf(Gamma( a, 1.0), b)` with a unity rate parameter.